### PR TITLE
Added pathes and definition for api logs

### PR DESCRIPTION
**Pull request by maxyengine**

### CODE REVIEW

**Comment by slavcodev** [comment]
In this repository, was changed file name convention for `paths`  definitions. We should use `@` instead of `.` (dot)

File: spec/definitions/ApiLog.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    type: string
+    enum:
+      - HEAD
+      - GET
+      - POST
+      - PUT
+      - DELETE
+    readOnly: true
+  ipAddress:
+    description: Request IP address
```
Comment (slavcodev) [inline spec/definitions/ApiLog.yaml]: It's `Client` IP address?

Comment (maxyengine) [inline spec/definitions/ApiLog.yaml]: yes

**Comment by slavcodev** [comment]
You're using tag `Tracking`, but it's not exists. Please also add it definition.

**Comment by slavcodev** [comment]
:+1:

**Pull request merged**